packages/jellyfish-api-core/__tests__/category/loan/loan_container.ts

```diff
@@ -2,7 +2,7 @@ import { MasterNodeRegTestContainer, StartOptions, GenesisKeys, MasterNodeKey }
 
 export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
   constructor (masternodeKey: MasterNodeKey = GenesisKeys[0]) {
-    super(masternodeKey, 'defi/defichain:HEAD-fffc8dd')
+    super(masternodeKey, 'defi/defichain:HEAD-5c9779c')
```

Comment:
can try the latest one?

```suggestion
    super(masternodeKey, 'defi/defichain:HEAD-39c3669')
```